stable/database/files/nuosm

```diff
@@ -1,22 +1,31 @@
-#!/bin/sh
+#!/bin/bash
 
-[ -n "$NUOSM_LINT" ] && set -n
+[ "$NUODB_DEBUG" = "verbose" ] && set -x
```

Comment:
I couldn't find if this is set by default or not. It might be better to have all debug messages printed by default for easier troubleshooting.